### DISCUSSION TITLE
github.com/go-kratos/kratos/v2/metrics

### DISCUSSION THREAD

**Discussion by Sejenew** [go-kratos/kratos#3419] [General]
github.com/go-kratos/kratos/contrib/metrics/prometheus/v2 imports
github.com/go-kratos/kratos/v2/metrics: module github.com/go-kratos/kratos/v2@latest found (v2.8.0), 
but does not contain package github.com/go-kratos/kratos/v2/metrics

**Comment by 18042542** [comment]
github.com/go-kratos/kratos/v2/metrics: cannot find module providing package github.com/go-kratos/kratos/v2/metrics

**Reply by 18042542** [reply]
go: module github.com/go-kratos/kratos/v2@upgrade found (v2.8.0), but does not contain package github.com/go-kratos/kratos/v2/metrics
